test/unit/MessageComposer/middleware/pollComposer/state.test.ts

```diff
@@ -182,24 +182,36 @@ describe('PollComposerStateMiddleware', () => {
       expect(result.status).toBeUndefined;
     });
 
-    it('should add a new empty option when the last option is filled', async () => {
+    it('should add a new empty option when the all the options are filled', async () => {
       const stateMiddleware = setup();
       const result = await stateMiddleware.handlers.handleFieldChange(
         setupHandlerParams({
           nextState: { ...getInitialState() },
```

Comment:
The next state has one empty option and the previous state has 3 options. That is inconsistent.

Reply:
Maybe you could keep this test as it was and add a new tests that:

1. simulates the scenario of moving an option to a different position
2. simulates the scenario of providing a value to a previously empty non-last option

Reply:
I have pushed the suggestions. Please let me know what you think. Thanks 😄